controller/argument_value_resolver.rst

```diff
@@ -51,12 +55,12 @@ controller::
 
     namespace AppBundle\Controller;
 
-    use AppBundle\Entity\User;
     use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\Security\Core\User\UserInterface;
 
     class UserController
     {
-        public function indexAction(User $user)
+        public function indexAction(UserInterface $user)
```

Comment:
I think I figured it out. There are two issues. One is that it would be helpful to link to the ParamConverter page and clarify when that is appropriate. I'll update my pull request.
The other is a bug in the DoctrineParamConverter that is causing the current example to fail. I will report that bug too.